src/pages/journal/components/indicators/indicators.scss

```diff
@@ -1,8 +1,6 @@
 indicators {
   div {
-    display: flex;
-    justify-content: center;
-    width: 2em;
+    padding-left: 8px;
```

Comment:
Is there a case for storing these values in sass variables? We'll probably have this same amount of padding in various places so putting it in a variable might make sense.

Reply:
Good shout. In this instance I can see how I maybe could unify the layout on the welsh logo above and indicator below. Currently they both have 16px total padding. The welsh logo has 4 + 6 + 6, and the indicator icon has 4 + 4 + 4 + 8.